SabaCenter/Model/Repositories/HomeRepository.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  HomeRepository.swift
+//  SabaCenter
+//
+//  Created by Mohsen Hosseini on 5/27/18.
+//  Copyright © 2018 Saba. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+public class HomeRepository: IHomeRepository {
```

Comment:
This should actually be broken up into `ScheduleRepository` and `HadithRepository`. It's not 1:1 Repositories for each PageViewModel. Services can have Repositories if there's data to cache (i.e. the service is requesting data from a service)